README.md

```diff
@@ -1,3 +1,5 @@
+![GitHub CI](https://github.com/grid-parity-exchange/Prescient/workflows/GitHub%20CI/badge.svg?event=schedule)
```

Comment:
On other projects I've used the equivalent of "push" instead of "schedule" as the event.  That way it always reflects the latest head instead of waiting 8 hours.  Thoughts?

Reply:
So the reason for using "schedule" here is so we can see if there was a breaking upstream change (like we recently experienced), which alone push will not allow.

If we want the badge to reflect both, I think I'd need to separate out the push/schedule events from the PR events in GitHub workflows.